int8 robot_id
---
bool success
string message